tests/cpptests/test_cpp_llapi.cpp

```diff
@@ -820,3 +820,14 @@ TEST_F(LLApiTest, testScorer) {
   RediSearch_ResultsIteratorFree(it);
   RediSearch_DropIndex(index);
 }
+
+TEST_F(LLApiTest, testStopwords) {
+  const char *words[] = {"Redis", "Labs"};
+  RSIndex* index = RediSearch_CreateIndex("index", NULL);
+  RediSearch_CreateStopwordsList(index, words, 2);
```

Comment:
Should we make sure it is working as expected (not found) if we call
`RediSearch_StopwordsList_Contains` between line 826 and 827? (list should be empty?)
Same question also after line 832.

Reply:
Good idea.
At that point, we have the default list, so we expect `is` to return `1` and `Redis` to return `0`.